packages/swapper/src/safe-utils/index.ts

```diff
@@ -0,0 +1,37 @@
+import { isEvmChainId } from '@shapeshiftoss/chain-adapters'
+import axios from 'axios'
+
+import { ChainIdToSafeBaseUrl } from './constants'
+import type {
+  FetchSafeTransactionArgs,
+  SafeTransactionError,
+  SafeTransactionSuccess,
+  SafeTxInfo,
+} from './types'
+
+export const fetchSafeTransactionInfo = async ({
+  chainId,
+  maybeSafeTxHash,
```

Comment:
suggestion - based on usage, this can be required and `safeTxHash`. if we want to use `| undefined` in case that changes, seems like we should add an early return on `!safeTxHash`.

Reply:
disregard the usage comment, hook can have `undefined` still suggest early return on undefined though

Reply:
closing as we use `skipToken`, this is already type-safe and required in `fetchSafeTransactionInfo` - renamed to `safeTxhash` for clarity however! aba3b9d5f2